python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -96,14 +102,65 @@ def define_hermetic_runtime_toolchain_impl(
         hdrs = [":includes"],
         includes = [
             "include",
-            "include/python{major}.{minor}".format(**version_dict),
-            "include/python{major}.{minor}m".format(**version_dict),
+        ] + select({
+            _IS_FREETHREADED: [
+                "include/python{major}.{minor}t".format(**version_dict),
+                # FIXME @aignas 2024-11-05: the following looks fishy - should
+                # we have a config setting for `m` (i.e. DEBUG builds)?
+                "include/python{major}.{minor}tm".format(**version_dict),
```

Comment:
I think the m-suffix was actually removed, but I can't find the docs that talk about it. I swear I read about this when I was looking into wheel format names and such.

But yes, "tm" is fishy.

Reply:
```suggestion
```